react/PieChart/index.jsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+import { Chart as ChartJS, ArcElement, Tooltip } from 'chart.js'
+import { Doughnut } from 'react-chartjs-2'
+import { useTheme } from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
```

Comment:
nit: should we used


import { makeStyles } from '@material-ui/core/styles'
or
the one for cozy-ui?

Reply:
ça revient au même, le makeStyles de cozy-ui n'a pas vocation à être modifié